src/vimscript/pattern.ts

```diff
@@ -40,6 +40,7 @@ export class Pattern {
   public readonly direction: SearchDirection;
   public readonly regex: RegExp;
   public readonly ignorecase: boolean | undefined;
+  public readonly inselection: boolean | undefined;
```

Comment:
Make this `inSelection`, please

Reply:
Also, please make it a `boolean` and default to `false` in the constructor

Reply:
fixed